### ISSUE TITLE
[1.26.0] Version 1.26.0 turns off User sandbox scripting by default and it breaks React Native CI builds

### ISSUE THREAD

**Issue by cipolleschi** [CocoaPods/Xcodeproj#989]
Yesterday (27th of Oct, 2024) The release of xcodeproj 1.26.0 happened.

After the Release, the [CI pipelines of React Native](https://github.com/facebook/react-native/actions/runs/11536322062/job/32112555761#step:3:757) and several other projects in the Ecosystem started failing.

We identified the reason to be a change in the value set for the "User Script Sandboxing" settings.

As a patch, we are going to pin the gem version for React Native to `< 1.26.0`. However, it would be great if we can restore the original behavior.

**Comment by iameli** [comment]
For anyone like me whose CI environment upgraded underneath you and broke your RN builds, this gets the job done:
```
sudo gem uninstall xcodeproj -x --ignore-dependencies
sudo gem install xcodeproj -v 1.25.1
```

**Comment by amorde** [comment]
Sorry about this. It took me longer than expected to get a CocoaPods release out. 1.16.0 was just released which should address this.

**Comment by RubenBez** [comment]
I saw react native also released an update to resolve this

https://github.com/facebook/react-native/releases/tag/v0.76.1

**Comment by cipolleschi** [comment]
yeah, for safety, we decided to release with the version pinned. The reason is that we are not forcing everyone to be on the latest cocoapods/xcodeproj version and we don't want to end up in a configuration that is not supported.

**Comment by amorde** [comment]
I've released [1.27.0](https://github.com/CocoaPods/Xcodeproj/releases/tag/1.27.0) which reverts the breaking changes to the default build settings

**Comment by abdulowork** [comment]
On a tangent note: I am not sure it makes sense to enable [BUILD_LIBRARY_FOR_DISTRIBUTION](https://github.com/CocoaPods/Xcodeproj/commit/e08b95121487b064cae8ac6607542c19253440fa#diff-ff9a4d09c635f0b936a889cc315e58fb2d38bf772b1712ea795498def364621aL325). It is unlikely an application wants library evolution to be able enabled in Swift compilations and it might break compilations which otherwise worked fine before (as it did it our case)
